src/workloads/scale/LargeScaleModel.yml

```diff
@@ -161,6 +161,7 @@ Actors:
   - {Nop: true}
   - {Nop: true}
   - Duration: *Duration
+    MetricsName: Update
```

Comment:
These "MetricsName: Update" were my attempts to keep old metrics name for the phase because I made changes about to support per phase metric name for the MultiCollectionUpdate actor.
But I dont know why this test (and large_scale_parallel and large_scale_serial) shows up in my new build variant even without specifying "replica-1dayhistory-15gbwtcache" in AutoRun, see [this patch build](https://evergreen.mongodb.com/version/5ef23cb40305b93e1b259610). Any idea why?

Reply:
If you select genny_auto_tasks it will run this workload any time we have `mongodb_setup` set to `single`. If you only want your changed workload to run, you select `genny_patch_tasks`. Upcoming changes to parameterized builds in evergreen will hopefully make this less painful.

Reply:
Just for posterity, if you select genny_patch_tasks we'll always schedule all changed workloads for that variant even if they don't specify that variant in their autorun sections [as this test shows](https://github.com/mongodb/genny/blob/44ca5e0b99563fd1d3b261b5249f52763134c029/src/python/tests/auto_tasks_test.py#L154). I'm not convinced this is "right," but it only affects patch-builds and only downside is some wasted cycles and a false-purple. We can revisit this logic if this turns out to be a common thing.